tensorflow_addons/custom_ops/image/__init__.py

```diff
@@ -12,13 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-"""
-Text-processing ops.
-"""
+"""Image manipulation ops"""
 from __future__ import absolute_import
 from __future__ import division
 from __future__ import print_function
 
-# Skip Gram Sample
-from tensorflow_addons.text.python.skip_gram_ops import skip_gram_sample
-from tensorflow_addons.text.python.skip_gram_ops import skip_gram_sample_with_text_vocab
+# Transforms
+from tensorflow_addons.custom_ops.image.python.transform import transform
```

Comment:
Although It's convenient for user, I'm a little worried about this usage, which will initialize all sub-modules as soon as user import it. Perhaps `tf.export` is a better solution, but anyway, I think it's accepted for now. Thanks :-)

Reply:
Good point. IMO user experience is more important until it becomes a major issue, but I could be persuaded otherwise with some performance numbers